docs/reference/ocm_ocm-uploadhandlers.md

```diff
@@ -73,9 +73,9 @@ The following handler names are possible:
 
     sub namespace of the form <code>&lt;plugin name>/&lt;handler></code>
 
-  - <code>ocm/mvnArtifact</code>: uploading mvn artifacts
+  - <code>ocm/mavenArtifact</code>: uploading mvn artifacts
 
-    The <code>ocm/mvnArtifact</code> uploader is able to upload mvn artifacts (whole GAV only!)
+    The <code>ocm/mavenArtifact</code> uploader is able to upload mvn artifacts (whole GAV only!)
```

Comment:
Why are we renaming this? The tool (on command line) is: `mvn` and not maven

Reply:
I thought, `maven` is a better name to represent the technology in general.

Reply:
But I still think, the package management technology itself is called Maven.